client/state/selectors/test/get-network-sites.js

```diff
@@ -59,7 +59,7 @@ describe( 'getNetworkSites()', () => {
 		expect( getNetworkSites( state, 1 ) ).to.be.null;
 	} );
 
-	test( 'should return an empty array if no secondary sites exist', () => {
+	test( 'should return only the main site no secondary sites exist', () => {
```

Comment:
Typo in "only the main site IF no secondary..".

Reply:
Fixed.